Pod/Classes/BONTag.h

```diff
@@ -0,0 +1,69 @@
+//
+//  BONTag.h
+//  Pods
+//
+//  Created by Nora Trapp on 5/11/16.
+//
+//
+
+#import <Foundation/Foundation.h>
+#import "BONCompatibility.h"
+
+@protocol BONTextable;
+
+@interface BONTag : NSObject <NSCopying>
+
+@property (copy, nonatomic, readonly, BONNonnull) NSString *startTag;
+@property (copy, nonatomic, readonly, BONNonnull) NSString *endTag;
+@property (copy, nonatomic, readonly, BONNonnull) NSString *escapeString;
+@property (copy, nonatomic, readonly, BONNonnull) id<BONTextable> textable;
+
+- (BONNonnull instancetype)initWithStartTag:(BONNonnull NSString *)startTag endTag:(BONNonnull NSString *)endTag escapeString:(BONNonnull NSString *)escapeString textable:(BONNonnull id<BONTextable>)textable;
+- (BONNonnull instancetype)initWithTag:(BONNonnull NSString *)tag textable:(BONNonnull id<BONTextable>)textable;
+
+@end
+
+/**
+ *  Create a tag with a custom start/end/escape combination.
+ *
+ *  @param startTag     The start tag.
+ *  @param endTag       The end tag.
+ *  @param escapeString The escape string.
+ *  @param textable     The style to apply.
+ *
+ *  @return A @p BONTag instance representing the tag.
+ */
+CG_INLINE BONTag *BONCNonnull BONTagComplexMake(NSString *BONCNonnull startTag, NSString *BONCNonnull endTag, NSString *BONCNonnull escapeString, id<BONTextable> BONCNonnull textable)
+{
+    return [[BONTag alloc] initWithStartTag:startTag endTag:endTag escapeString:escapeString textable:textable];
+}
+
+/**
+ *  Create a tag using the default start and end formats and using \ as an escape character. <tag> </tag>
```

Comment:
On a new line:

```
Example:
 @code
// code text
<tagName>Normal String</tagName>
<tagName>String with \<escaped characters\></tagName>
@endcode
```

^ or whatever would demonstrate escaping properly - which characters do you need to escape? Are those cases tested?

Syntax via http://stackoverflow.com/questions/19168423/what-are-the-new-documentation-commands-available-in-xcode-5